docs/source/markdown/options/mount.md

```diff
@@ -31,7 +31,9 @@ Options specific to type=**volume**:
 - *U*, *chown*: *true* or *false* (default if unspecified: *false*). Recursively change the owner and group of the source volume based on the UID and GID of the container.
 
 - *idmap*: If specified, create an idmapped mount to the target user namespace in the container.
-  The idmap option supports a custom mapping that can be different than the user namespace used by the container.
+  The idmap option is only supported by Podman in rootful mode. The Linux kernel does not allow the use of idmaped file systems for not root users.
```

Comment:
Is the language incorrect here?

_for not root users_

Maybe it should be:

_for non-root users_
or
_for unprivileged users_

Reply:
Alright, I've changed  it to "unprivileged"